deutsches-archaologisches-institut.csl

```diff
@@ -251,37 +246,28 @@
       <text variable="locator"/>
     </group>
   </macro>
-  <macro name="container-title-short">
-    <group>
-      <choose>
-        <if type="article-journal">
-          <choose>
-            <if variable="container-title-short" match="none">
-              <text variable="container-title"/>
-            </if>
-          </choose>
-          <text variable="container-title-short"/>
-        </if>
-      </choose>
-    </group>
-  </macro>
   <macro name="date">
     <date variable="issued">
       <date-part name="year"/>
     </date>
   </macro>
   <macro name="place-date">
     <choose>
-      <if type="book thesis" match="any">
-        <text variable="publisher-place" prefix=" (" suffix=" "/>
+      <if type="book thesis chapter" match="any">
+        <choose>
+          <if match="none" is-numeric="edition">
+            <text value=" "/>
+          </if>
+        </choose>
```

Comment:
(and it would be good to check if the code above works as desired; I couldn't find any example references at the documentation link, and hand-wrote the blurb without testing)